common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -114,6 +114,18 @@ class CapaFields(object):
         help=_("Amount of time after the due date that submissions will be accepted"),
         scope=Scope.settings
     )
+    show_correctness = String(
+        display_name=_("Show Correctness"),
+        help=_("Defines when to show whether a learner's answer to the problem is correct. "
+               "Configured on the subsection.  A default value can be set in Advanced Settings."),
```

Comment:
@srpearce [`show_correctness` isn't editable for individual problems](https://github.com/open-craft/edx-platform/blob/53824c674f1c063704bac9fae644db74a8bfb6af/common/lib/xmodule/xmodule/capa_module.py#L219), only on the subsections, because (in a future PR) we need to be able to hide correctness for a whole subsection on the Progress page, and we can't do that efficiently if the setting can be changed per problem.  However, we needed to have the field here as well as in the [`InheritanceMixin`](https://github.com/edx/edx-platform/pull/14737/files#diff-5ec9a0a362b39d81a40a5b5d6a55bedcR104), so that the CAPA code could access it.

So the help text here isn't actually used.. I've removed the extra space before "A default value..", but can remove the entire help text if it's confusing/inaccurate.